hasher-matcher-actioner/scripts/storm_images_queue

```diff
@@ -108,19 +108,16 @@ if __name__ == "__main__":
     )
     parser.add_argument(
         "queue_url",
-        type=str,
```

Comment:
What are these changes for?

Reply:
Was in the area and they are nitty changes (str is the default type if unspecified). If you want to be ultra strict I can move them to another commit